cli/schemas/environment/environment.go

```diff
@@ -47,6 +50,12 @@ type Docker struct {
 
 //Validate returns an error for invalid environment.yml files
 func (e *Environment) Validate() error {
+	if e.Name == "" {
```

Comment:
@rberrelleza this is a hard constraint we are adding in this PR.
I want to have the environment name in the Cloud Formation stack name, which only allows alphanumeric and hyphens. In order to avoid duplicates, service and environment names can only have alphanumerics.